server2/Arcadia.Assistant/Arcadia.Assistant.ExternalStorages.Abstractions/Arcadia.Assistant.ExternalStorages.Abstractions.csproj

```diff
@@ -0,0 +1,13 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.1</TargetFramework>
+    <Platforms>AnyCPU;x64</Platforms>
```

Comment:
There are some projects with overrided default settings where not needed. Could you fix them all?

Reply:
It's Alex changes. Hi approved it. Please, discuss it with him if you have another opinion.

Reply:
I guess, he didn't notice that these changes differ from default. @santee?